src/plugins/dashboard/common/saved_dashboard_references.ts

```diff
@@ -24,7 +23,7 @@ export interface SavedObjectAttributesAndReferences {
 }
 
 const isPre730Panel = (panel: Record<string, string>): boolean => {
-  return 'version' in panel ? semverSatisfies(panel.version, '<7.3') : true;
+  return 'version' in panel ? Semver.gt('7.3.0', panel.version) : true;
```

Comment:
noticed that migration spent about 7.4sec here, with these changes, migration is `0.3sec. There are other `semverSatisfies` usages in `dashboard` plugin, so it's up to code-owners whether to refactor all the usages. 

before:
![Screenshot 2021-05-11 at 14 55 09](https://user-images.githubusercontent.com/3198181/117818657-1cd58e80-b269-11eb-82e0-85b950224521.png)

after:
<img width="377" alt="Screenshot 2021-05-11 at 14 50 37" src="https://user-images.githubusercontent.com/3198181/117818534-fadc0c00-b268-11eb-9d8f-7147034c8940.png">

Reply:
Not surprising given that `<7.3` needed to be parsed and interpreted.